terraform/aws/etc/eu-west-2_vp.tfvars

```diff
@@ -66,18 +66,13 @@ gpc-consumer_service_container_port = 8080
 gpc-consumer_service_launch_type = "FARGATE"
 gpc-consumer_root_log_level = "WARN"
 gpc-consumer_log_level = "INFO"
-gpc-consumer_logging_format = "(*)"
-gpc-consumer_mesh_host = "hhttps://mesh.vp.nhsredteam.internal.nhs.uk:8829/messageexchange/"
-gpc-consumer_mesh_cert_validation = "true"
-gpc-consumer_mongo_options = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
-gpc-consumer_ssl_trust_store_url = "s3://nhsd-aws-truststore/rds-truststore.jks"
+gpc-consumer_logging_format = ""
 gpc-consumer_logs_datetime_format = "%Y-%m-%d %H:%M:%S%L"
-gpc-consumer_create_wiremock = false
-gpc-consumer_wiremock_container_port = 8080
-gpc-consumer_wiremock_application_port = 8080
 gpc-consumer_sds_url = "https://sandbox.api.service.nhs.uk/spine-directory/"
```

Comment:
this should be wiremock, no sandbox for vp

Reply:
maybe just set to empty string for now and we'll pick it up again closer to running vp tests. I just don't want us to run the tests against sandbox accidentally

Reply:
Its a default value to be used if GP2GP-Wiremock is not enabled in environment. It gets changed to wiremock URL if wiremock is enabled. see following;

[https://github.com/nhsconnect/integration-adaptors/blob/0a2da08a2d25136e61a1f9b9d47e67c10727cb7e/terraform/aws/components/gp2gp/gpc-consumer-locals_env_variables.tf#L24](https://github.com/nhsconnect/integration-adaptors/blob/0a2da08a2d25136e61a1f9b9d47e67c10727cb7e/terraform/aws/components/gp2gp/gpc-consumer-locals_env_variables.tf#L24)